ert_shared/libres_facade.py

```diff
@@ -240,3 +243,16 @@ def get_workflow_job(self, name):
         if self._enkf_main.getWorkflowList().hasJob(name):
             return self._enkf_main.getWorkflowList().getJob(name)
         return None
+
+    def get_ensemble_parameters(self, ensemble_id: UUID) -> List[dict]:
```

Comment:
should this code handle the `ensemble_id` at some point, or what is the purpose?

Reply:
I guess in the context of enkf main you will only be able to get info for the current ensemble so it makes sense to remove the id. Same for comment below.

Reply:
I guess at least for the time being we could have that assumption. But i'm not sure how to handle this in the future visualizing all ensembles - we probably need to introduce some kind of magic :man_shrugging: